packages/ui-components/src/components/pagination/index.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
```

Comment:
should be renamed to **index.ts** and export the whole file. prefer view on the navigation index.ts file @fallingforest